src/components/Banner.less

```diff
@@ -0,0 +1,117 @@
+@gray-40: #b4b5b6;
+@gray-5: #f7f8f9;
+
+.m-global-eyebrow {
+  font-size: 0.75em;
```

Comment:
With the DS imported, you should be able to copy over the full [cf.gov eyebrow file](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/molecules/global-eyebrow.less) BUT we're using a newer version of Less so you'll have to add some parens around the first argument. e.g. Instead of

```less
  font-size: unit(12px / @base-font-size-px, em);
```

it'd be

```less
  font-size: unit((12px / @base-font-size-px), em);
```

Reply:
Perens!! Updated.  I didn't see these errors previously because I was importing the less variables incorrectly, so thanks for the guidance here.